src/argilla/server/alembic/versions/7cbcccf8b57a_create_metadata_properties_table.py

```diff
@@ -33,12 +33,14 @@ def upgrade() -> None:
     # ### commands auto generated by Alembic - please adjust! ###
     op.create_table(
         "metadata_properties",
+        sa.Column("id", sa.Uuid(), nullable=False),
         sa.Column("name", sa.Text(), nullable=False),
+        # TODO: We should move type column to settings as an attribute (as we are already doing for Fields and Questions).
         sa.Column("type", sa.Text(), nullable=False),
-        sa.Column("description", sa.Text(), nullable=True),
+        sa.Column("description", sa.Text()),
```

Comment:
why have we removed the `nullable=True`?

Reply:
This should be fixed in the next PR with the new endpoint to update metadata properties.